frontend/src/metabase/dashboard/components/DashCard/DashCardTabMenu/DashCardTabMenu.tsx

```diff
@@ -23,6 +27,22 @@ export function DashCardTabMenu({ dashCardId }: DashCardTabMenuProps) {
     return tabs.filter(t => t.id !== selectedTabId);
   }, [selectedTabId, tabs]);
 
+  const moveDashcard = useCallback(
+    (destinationTabId: number) => {
+      dispatch(moveDashCardToTab({ dashCardId, destinationTabId }));
+      dispatch(
+        addUndo({
+          message: t`Card moved`,
```

Comment:
@cdeweyx @dhuniverse Not sure if this is the most updated toast message as the previous version isn't this one
```ts
t`Card moved to ${tabs.find(c => c.id === destTabId)?.name}`
```
But [it matches the spec](https://www.notion.so/metabase/Let-people-move-cards-between-dashboard-tabs-e797b321092b45a1bcbc4478d982a01e?pvs=4#1b2b364225a7477999c2b62a29b15e8d)
![Screenshot 2023-10-30 at 1 15 59 PM](https://github.com/metabase/metabase/assets/1937582/398cbbbc-a31b-4bdc-8598-1e9b63bf7825)


I found that this is very confusing when you're moving multiple cards between tabs. I'm not sure how we could make this better. But maybe something like showing the type of the card or the card name would make it clearer. e.g. **Text card moved**, **Link card moved**, **Heading card moved**, **Question moved** (or even the name of the question itself)

What do you think?

Reply:
Yeah, let's leave this as is until we hear from product since this is already matching the spec.

Reply:
Good feedback, I'll start a thread to discuss with design. As for now feel free to merge and we can improve this case in M4.